spyder/plugins/projects.py

```diff
@@ -153,6 +154,9 @@ def register_plugin(self):
         self.sig_project_closed.connect(
             lambda v: self.editor.setup_open_files())
         self.recent_project_menu.aboutToShow.connect(self.setup_menu_actions)
+        self.run.connect(lambda fname: self.main.open_external_console(

```

Comment:
I think we don't have an option to run a file on a dedicated console: [app/mainwindow.py#L2423](https://github.com/spyder-ide/spyder/blob/master/spyder/app/mainwindow.py#L2423)